spel/scripts/amigen7-build.sh

```diff
@@ -340,12 +401,12 @@ do
     ln "${RPM}" "${ELBUILD}"/AWSpkgs/
 done
 
-echo "Executing DiskSetup.sh"
-bash -eux -o pipefail "${ELBUILD}"/DiskSetup.sh -b "${BOOTLABEL}" -v "${AMIGENVGNAME}" -d "${AMIGENBUILDDEV}" -p "${AMIGENSTORLAY}" -B "${AMIGENBOOTSIZE}" || \
+# Invoke disk-partitioner
+bash -euxo pipefail "${ELBUILD}"/"$( ComposeDiskSetupString )" || \
```

Comment:
I think composing the string this way is running afoul of string interpolation issues...

```
minimal-centos-7-hvm: + bash -euxo pipefail '/tmp/el-build/DiskSetup.sh -B 500m -b /boot -f ext4 -p /:rootVol:4,swap:swapVol:2,/home:homeVol:1,/var:varVol:2,/var/log:logVol:2,/var/log/audit:auditVol:100%FREE -v VolGroup00 -d /dev/nvme0n1'
```

Looks like the scriptname and all its args are being passed within quotes, so the shell treats them all as a single argument to the command, `bash`. i.e. `bash 'script arg1 arg2'` instead of `bash 'script' 'arg1' 'arg2'`...